### ISSUE TITLE
Add log location option

### ISSUE THREAD

**Issue by 43615** [ArtificialQualia/PyEveLiveDPS#91]
I'm playing on Steam through Proton, which puts the logs under `.../SteamLibrary/steamapps/compatdata/8500/pfx/drive_c/users/steamuser/Documents/EVE/logs/Gamelogs/`. There appears to be no option for changing the directory anywhere. I can only play back existing logs by manually selecting them.

**Comment by ArtificialQualia** [comment]
Makes sense, I'll see about adding this when I get the chance.  Others are also welcome to open a PR.
